modules/olm-installing-specific-version-cli.adoc

```diff
@@ -30,6 +30,110 @@ endif::[]
 
 .Procedure
 
+. Run following command to look up the available channels and versions of an Operator that can be installed on your system:
++
+.Command syntax
+[source,terminal]
+----
+$ oc get packagemanifests <operator_name> -n <catalog_namespace> -o <output_format>
+----
++
+For example, the following command prints the available channels and versions of the Red Hat Quay Operator from the OperatorHub in the YAML format:
++
+.Example command
+[source,terminal]
+----
+$ oc get packagemanifests quay-operator -n openshift-marketplace -o yaml
```

Comment:
@awgreene Does it make sense to use `oc describe packagemanifests` to remain consistent with step two in the procedure above? [Installing from OperatorHub using the CLI](https://57833--docspreview.netlify.app/openshift-enterprise/latest/operators/admin/olm-adding-operators-to-cluster.html#olm-installing-operator-from-operatorhub-using-cli_olm-adding-operators-to-a-cluster)

Reply:
Also, WDYT about putting the feature in this procedure vs. as a one step procedure in a separate module? 

To put it another way, does it the place where a customer would need this information?

Reply:
Should be fine either way as long as it is consistent.